pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -444,9 +444,34 @@ func (c *Check) StatementMetrics() (int, error) {
 		var diff OracleRowMonotonicCount
 		planErrors = 0
 		for _, statementMetricRow := range statementMetricsAll {
+
+			var trace bool
+			for _, t := range c.config.QueryMetrics.Trackers {
+				if len(t.ContainsText) > 0 {
+					for _, q := range t.ContainsText {
+						if strings.Contains(statementMetricRow.SQLText, q) {
+							trace = true
+						} else {
+							trace = false
+							break
+						}
+					}
+					if trace {
+						break
+					}
+				}
+			}
+
+			if trace {
+				log.Infof("%s queried: %+v", c.logPrompt, statementMetricRow)
```

Comment:
if we don't want to expose this feature to all end users, should this to be `debug` level so the log lines aren't always visible? i don't have a deep understanding of the use case for this so possibly this suggestion makes no sense.

Reply:
The purpose of the feature is to get the insight into the query metrics processing by selecting only the representative subset of the payload with minimal verbosity. For example, a customer noticed inflated metrics on several `insert` statements. In such case, we would configure the tracking on one of those `inserts` to get the data needed to analyze the issue without having to trace the whole payload which might be huge. The information written into the log would be minimal.

With "not exposing this feature to all end users", I meant not documenting it in `conf.yaml.example` so people wouldn't start configuring it by themselves. But if a customer reports a problem with query metrics, we would give them instructions how to configure tracking, so they would expect some temporary `INFO` messages in the log file. 

If we set `log_level` to `DEBUG`, the agent would start logging whole payloads and clutter the log file.

Let me know if you still think, that we should log tracking as `DEBUG`.

Reply:
Ah ok, thanks for the additional context. This makes sense to me.